mods/cnc/weapons/missiles.yaml

```diff
@@ -178,15 +122,11 @@ MammothMissiles:
 	Report: rocket1.aud
 	ValidTargets: Ground
 	Projectile: Bullet
-		Arm: 5
-		Blockable: false
 		Shadow: true
 		Inaccuracy: 853
 		LaunchAngle: 62
 		Image: DRAGON
-		RateOfTurn: 2
```

Comment:
This is replaced by `HorizontalRateOfTurn: 15` via the inherits.  Is this what we want?